Porting/bisect-runner.pl

```diff
@@ -86,7 +86,24 @@
 
 my ($target, $match) = @options{qw(target match)};
 
-@ARGV = ('sh', '-c', 'cd t && ./perl TEST base/*.t')
+# El Capitan (OS X 10.11) (and later) strip DYLD_LIBRARY_PATH
+# from the environment of /bin/sh
+# https://developer.apple.com/library/archive/documentation/Security/Conceptual/System_Integrity_Protection_Guide/RuntimeProtections/RuntimeProtections.html
+#
+# (They *could* have chosen instead to ignore it and pass it through. It would
+# have the same direct effect, but maybe needing more coding. I suspect the
+# choice to strip it was deliberate, as it will also eliminate a bunch more
+# attack vectors, because it prevents you sneaking an override "into" something
+# else you convince the user to run.)
+
+my $agressive_apple_security = "";
```

Comment:
Spelling: shouldn't this be: `$aggressive_apple_security` (here and below)?

Reply:
D'oh yes!

Well spotted. Thanks. Will fix.

Reply:
@nwc10, could you make the requested spelling correction?  I'd be willing to +1 the p.r. thereafter.  Thanks.

Reply:
> @nwc10, could you make the requested spelling correction? I'd be willing to +1 the p.r. thereafter. Thanks.

@nwc10, ping

Reply:
@nwc10 - don't worry about it. I took care of it for you.

Reply:
@jkeenan ping. Can you please test the rebased version of this branch?